app/build.gradle

```diff
@@ -59,29 +59,29 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+        sourceCompatibility JavaVersion.VERSION_17
```

Comment:
Since we are imposing Java 17 or later, I think that the `README.md` must updated to reflect this change.

Reply:
Okay, I think I corrected it, I decided to remove the mention of Android Studio altogether.